Automattic/woocommerce-payments
# Remove redundant OTP events

The Tracks events in the WooPay OTP flow recorded from the merchant site are no longer needed. We have equivalent events recorded from WooPay so these events can now be removed.

#### Changes proposed in this Pull Request

Tracks events removed:
```
woocommerceanalytics_woopay_otp_prompt_start
woocommerceanalytics_woopay_otp_prompt_complete
woocommerceanalytics_woopay_otp_prompt_failed
```








#### Testing instructions





* Add a product to the cart
* As a registered WooPay user, initiate the OTP auth flow (make sure the OTP model shows up)
* Open network tab in browser dev tools
* Enter an invalid OTP code
* Enter the correct OTP code
* There should not be any `admin-ajax` requests with above event names under the `tracksEventName` field.
* Repeat the tests using the WooPay button on cart and product pages

-------------------


**Post merge**
